README.md

```diff
@@ -48,19 +48,34 @@ The `master` branch tracks the ongoing efforts towards the next stable release t
 
 ## Quick Install
 
+**Linux / Mac**
+
 By using the installer to install the latest stable version:
 ```bash
 bash <(curl -s https://install.aeternity.io/install.sh)
 ```
+See the documentation for [starting](https://docs.aeternity.io/en/stable/operation/#start-the-node) and [configuring](https://docs.aeternity.io/en/stable/configuration/#example) the node.
+
+## Docker
+
+Alternatively, you can run the node client as a docker container:
+
+**Linux / Mac**
 
 Or running a docker container (latest tag):
 ```bash
-mkdir -p ~/.aeternity/maindb
-docker pull aeternity/aeternity
-docker run -p 3013:3013 -p 3015:3015 \
+$ mkdir -p ~/.aeternity/maindb
+$ docker pull aeternity/aeternity
+$ docker run -p 3013:3013 -p 3015:3015 \
     -v ~/.aeternity/maindb:/home/aeternity/node/data/mnesia \
     aeternity/aeternity
 ```
+**Windows**
+```bash
+$ mkdir %APPDATA%\aeternity\maindb
+$ docker pull aeternity/aeternity
+$ docker run -p 3013:3013 -p 3015:3015 -v %APPDATA%/aeternity/maindb:/home/aeternity/node/data/mnesia aeternity/aeternity
```

Comment:
please make it multi-line to be consistent with the above command

Reply:
no, because in command prompt and powershell the newline escape sign is inconsistent, we'll keep it simple.